Dedicated pricing page with plan comparison

Lindharden opened the pull request.

flopper123 reviewed (changes requested):
Overall looks super nice! I have two small comments:

1. The pills on light mode are a little invisible, at least on my screen. Maybe they should have a border of some kind
<img width="1186" height="328" alt="image" src="https://github.com/user-attachments/assets/40694d2f-0185-452d-8019-e41fc681eb26" />

2. The "Pricing" link in the public top bar should probably link to the new page:
<img width="1072" height="431" alt="image" src="https://github.com/user-attachments/assets/cf13754f-7bfe-4f11-96b2-accda1f8d90d" />

3. I get an error saying there's are duplicate keys:
<img width="1090" height="465" alt="image" src="https://github.com/user-attachments/assets/d1e990f5-bf2e-4896-80ae-371a69800a55" />

Lindharden commented:
> Overall looks super nice! I have two small comments:
> 
> 1. The pills on light mode are a little invisible, at least on my screen. Maybe they should have a border of some kind

That is delibarate. Border does not look nice. They intentionally appear if almost just placed directly on the background :)

> 2. The "Pricing" link in the public top bar should probably link to the new page:

Nice catch, fixed 👍 

> 3. I get an error saying there's are duplicate keys:

Fixed 👍

flopper123 reviewed (approved):
Awesome! 🤩
